interface/modules/custom_modules/oe-module-claimrev-connect/table.sql

```diff
@@ -0,0 +1,32 @@
+-- This table definition is loaded and then executed when the OpenEMR interface's install button is clicked.
+CREATE TABLE IF NOT EXISTS `mod_claimrev_eligibility`(
+    `id` INT(11)  PRIMARY KEY AUTO_INCREMENT NOT NULL
+    ,`pid` bigint(20)
+    ,`payer_responsibility` varchar(2)
+    ,`request_json` TEXT
+    ,`response_json` TEXT
+	,`eligibility_json` TEXT
+	,`individual_json` TEXT
+    ,`response_message` TEXT
+    ,`status` varchar(25)
+    ,`last_checked` datetime
+    ,`create_date` datetime
```

Comment:
2 above datetime entries should default to NULL

Reply:
I had null, and the script failed on installing so I took it out.

Reply:
Would change 2 above lines (should definitely work) to the following (need to have default NULL since OpenEMR is moving to standard sql-mode that requires NULL or an actual date/datetime/timestamp.
```
    ,`last_checked` datetime default NULL
    ,`create_date` datetime default NULL
```